cli/src/semgrep/meta.py

```diff
@@ -659,6 +670,11 @@ def repo_name(self) -> str:
             return project_path
         return super().repo_name
 
+    @property
+    def repo_display_name(self) -> str:
+        repo_display_name = os.getenv("SEMGREP_REPO_DISPLAY_NAME", self.repo_name)
+        return repo_display_name or self.repo_name
```

Comment:
probably cleaner to have all of these (besides the one in the GitMeta class) call the super().repo_display_name method, which will keep this logic centralized in one place. Then the rest of these classes will inherit that behavior from GitMeta 👍 

```suggestion
    @property
    def repo_display_name(self) -> str:
        return super().repo_display_name
```